core/src/main/java/com/google/errorprone/bugpatterns/time/ZoneIdOfZ.java

```diff
@@ -57,7 +57,7 @@ public final class ZoneIdOfZ extends BugChecker implements MethodInvocationTreeM
   public Description matchMethodInvocation(MethodInvocationTree tree, VisitorState state) {
     if (ZONE_ID_OF.matches(tree, state)) {
       String zone = constValue(tree.getArguments().get(0), String.class);
-      if (zone != null && zone.equals("Z")) {
+      if (zone != null && (zone.equals("Z") || zone.equals("UTC"))) {
```

Comment:
So technically, `ZoneId.of("UTC")` is not equal to `ZoneOffset.UTC`. They have different values for `getId()`, (the former is `"Z"` and the latter is `"UTC"` I believe)...whether this matters in practice, I'm not sure.

Any ideas?

Reply:
Yea let's revert this to be on the safe side, especially since this is enabled as an error, not just a warning.